draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1439,159 +1538,248 @@ Steps:
 6. return CMOV(sign, 1, sign == 0)      # Regard x == 0 as positive
 ~~~
 
-# Hashing to a Finite Field {#hashtobase}
+# Hashing to a Finite Field {#hashtofield}
 
-The hash\_to\_base function hashes a string msg of any length into an element of a
-field F. This function is parametrized by the field F ({{bg-curves}}) and by H,
-a cryptographic hash function that outputs b bits.
+The hash\_to\_field function hashes an octet-string msg of any length into
+one or more elements of a field F.
+This function works in two steps: it first expands the input octet-string
```

Comment:
`it first expand the input` 
the first time I read this sentence, it gives the idea that if I have a large message to be hashed, then the hash_to_base function will expand to a bigger string. 
Actually, what happens is the opposite, the input message is hashed to produce a pseudo-random string. 
Maybe we can rephrase this sentence.